File: Data Structures & Algorithms/remove-duplicates-from-sorted-array/submission-0.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int  write = 1;
        for(int i =0;i<nums.size()-1;i++){
            if(nums[i]!=nums[i+1]){
                nums[write] = nums[i+1];
                write++;
            }
            else{
               continue;
            }
        }
        return write;
    }
};

File: Data Structures & Algorithms/two-integer-sum/submission-0.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        hasmap = collections.defaultdict(int)
        for i, num in enumerate(nums):
            diff = target - num
            if diff in hasmap:               
                return [hasmap[target-num],i]
            hasmap[num] = i
        return False
        